_build/objectEditingOutput/WurstExportedObjects_w3a.wurst.txt

```diff
@@ -1,778 +0,0 @@
-package WurstExportedObjects_w3a
```

Comment:
Don't delete this, it's useful for debugging, and for observing balance changes between versions.